src/store/project.ts

```diff
@@ -82,6 +82,16 @@ export class Project {
 		return this.id;
 	}
 
+	// TODO
+	public getLastChangedAuthor(): string {
+		return 'Max Mustermann';
+	}
+
+	// TODO
+	public getLastChangedDate(): Date {
```

Comment:
I don't see that function will make sense at all, at least in the moment.
We will need to prepare more to make that work, which is also not planned at the moment. That's why I would delete that code too, it's dead and will not be used in the near future